sogs/routes/rooms.py

```diff
@@ -58,6 +66,65 @@ def get_rooms():
     return jsonify([get_one_room(r) for r in mroom.get_readable_rooms(g.user)])
 
 
+BAD_NAME_CHARS = {c: None for c in range(32)}
+BAD_DESCRIPTION_CHARS = {c: None for c in range(32) if not (0x09 <= c <= 0x0A)}
+
+
+@rooms.put("/room/<Room:room>")
+@auth.user_required
```

Comment:
feel like we could have a `@auth.require_admin` for this

Reply:
I'm not sure how to make that work through a decorator, because it isn't just global admins but also room admins (which means we need to be able to call `room.check_admin(g.user)`, and so need the `room` argument).

Reply:
Added.